
 I'm searching for passengers similar to Myself:
 - Male
 - 16-18
 - Would likely travel 3rd class
 - Traveling with family
 - Lower fare

In [10]:
import sqlite3
import pandas as pd

# Connect to database
conn = sqlite3.connect('titanic.db')
print("✅ Connected to database")

✅ Connected to database


In [13]:
query = """
    SELECT Name, Age, Sex, Pclass, Survived, SibSp, Fare
    FROM passengers
    WHERE Sex = 'male' AND Age >=16 AND Age <= 18
        AND Pclass = 3 AND Survived = 1
    ORDER BY Fare ASC
    LIMIT 10;
"""

result = pd.read_sql(query, conn)
display(result)

name   age   sex  pclass  survived  sibsp  fare
0        Cohen, Mr. Gurshon "Gus"  18.0  male       3         1      0  8.05
1  Sunderland, Mr. Victor Francis  16.0  male       3         1      0  8.05

I PICK GUS:
Gus is cool, just like me. They have low fares, and are in 3rd class, just like me. They are also men, and I'm a man, so that matches pretty well. There's no other  individuals that are my age that payed a low fare other than Victor, who has the same other statistics but is younger.
The most important factors are (Which left me with two options):
Age: 18.0
Sex: Male
Pclass: 3
Survived: 1


My Report on Mr. Gurshon "Gus" Cohen:

Mr. Gurshon "Gus" Cohen stands out as one of the few third-class passengers who managed to survive the sinking of the Titanic. At 18 years old, he was a young man whose demographic and socio-economic profile closely matched the specific criteria I used in my search. He paid just $8.05 for his journey, representing the common financial experience of those traveling in third class. I chose to focus on Gus because he shared my age, gender, and economic status, making his survival story feel relevant to me. Ultimately, he represents the most accurate match for my own profile among those found in the database. 

Encyclopedia Titanic states that Gus was born in London, England, on December 31, 1892. He was Jewish and was part of a large family that moved to London from the Russian Area. He had a printing business, and due to its failure in London, he planned to cross over to the Americas and unite with his Uncle, who had a business in New York. He boarded the Titanic with money borrowed from a relative for £8, getting a low third-class seat, where he roomed with 5 other men. At the time of the collision, he was asleep, but the sound of the collision woke him, so he got up to find out what had happened. After putting on his life jacket, he climbed up to the higher decks to try and board a lifeboat. He was refused by multiple, because they were full or had priority riders. The matter of his survival is questionable, as he wasn't registered to have gotten on a lifeboat before the Titanic was sunk. He was probably able to sneak on one, or one picked him up after the ship initially sank. 

All data found in the database was confirmed by encyclopedia-titanica. Gus earned the nickname "The Cat" because he survived so many brushes with death. After surviving the Titanic, he fought in WWI where he was shot twice and lost an eye. Over the following years, he survived falling onto train tracks, a hit-and-run accident, and a deadly illness. He even made it through a bombing during WWII. During the 1950s, Gus communicated with various writers and several televised interviewers sharing his narrow escape from death.

Work Cited:
Encyclopedia Titanica (2018) Gurshon Cohen ref: #725, last updated: 30th June 2018, accessed 7th January 2025.
URL : https://www.encyclopedia-titanica.org/titanic-survivor/gurshon-gus-cohen.html

In [ ]:
## Part 4: Reflection Questions
1. **Database Skills:** How did using SQL help you narrow down passengers to find your match? What query challenges did you face?
        The only query challenge I faced was not knowing what to do at first. But once I knew, SQL helped me find the passenger I was looking for.

2. **Research Process:** What was the most difficult part of researching your passenger? What made certain sources more reliable than others?
        Finding him on encyclopedia titanica. Once I found him, it gave me all the information I needed. 
                                                                                                                                             
3. **Historical Connection:** What surprised you most about your passenger's story? How did learning about a real person change your perspective on the Titanic disaster?
        I learned that this dude is invincible. He survived many encounters with death. Learning about a real survivors story made the titanic disaster feel more real.

4. **Data vs. Stories:** The database shows numbers and categories, but your research revealed a real person's story. How did these two types of information work together to give you a fuller picture?
        Well the story was supported by the data. The numbers describe and narrow down passengers, while the information from research gives the bigger picture and provides character to the passenger.

5. **Information Literacy:** In an age of AI and quick answers, why is it important to find and evaluate original sources yourself?
        AI is bad at evaluating sources and only spits out the most accessable information. Evaluating sources yourself helps to find reliable data. 
